index.js

```diff
@@ -0,0 +1,66 @@
+function shuffle_array(array) {
+    for (let index = array.length - 1; index > 0; index--) {
+        let random_index = Math.floor(Math.random() * (index + 1));
+        let temp = array[index];
+        array[index] = array[random_index];
+        array[random_index] = temp;
+    }
+    return array
+}
+
+function get_new_question() {
+    if (remaining_movies.length < 1){
+        let user_feedback_element = document.querySelector('#user_feedback');
+        user_feedback_element.innerHTML = '<h3>You have run out of questions</h3>';
+    } else {
+        let current_movie = remaining_movies.pop();
+        let quote_element = document.querySelector('#quote');
+        quote_element.textContent = current_movie.quote;
+
+        let answer_element = document.querySelector('#answer');
+        answer_element.style.display="none";
```

Comment:
Is this answer element ever actually going to be displayed? If not I would suggest setting it to display none in the CSS rather than here. 

Is there actually a need to have the answer in the DOM or could you store it in a variable instead? Makes it slightly harder for a tech-savvy user to cheat and find the answer!

Reply:
Yes it actually being used.